src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -178,43 +178,63 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
   }
 
   fetchCounts = async () => {
-    const { allowedTypes } = this.props;
+    const { allowedTypes, namespaceRegistry } = this.props;
     const { queryText, visibleTypes, visibleNamespaces } = parseQuery(this.state.activeQuery);
 
     const filteredTypes = allowedTypes.filter(
       (type) => !visibleTypes || visibleTypes.includes(type)
     );
 
+    let availableNamespaces = namespaceRegistry.getAll() || [];
+    availableNamespaces = availableNamespaces.map((ns) => ns.id);
+
+    const filteredCountOptions = {
+      typesToInclude: filteredTypes,
+      searchString: queryText,
+    };
+
+    if (availableNamespaces?.length) {
+      if (visibleNamespaces?.length) {
+        filteredCountOptions.namespacesToInclude = visibleNamespaces;
```

Comment:
Based on this it looks like namespaces might not be set if available namespaces is empty, but the options param in getSavedObjectCountshas it marked as mandatory.

Reply:
@ashwin-pc The options for `getSavedObjectCounts` are:

```
interface SavedObjectCountOptions {
  typesToInclude: string[];
  namespacesToInclude?: string[];
  searchString?: string;
}
```

The question mark after `namespacesToInclude` indicates that this is optional right?

FYI I re-factored this area to make sure that any namespace listed in `visibleNamespaces` (these are the selected namespaces from the filter or from the search bar) is also in `availableNamespaces` (this is the list of registered namespaces and the one the user has access to). I think the block should read clearer now.

Reply:
Oops, sorry. I must have seen something else.